nautobot/utilities/filters.py

```diff
@@ -114,7 +114,7 @@ class MultiValueMACAddressFilter(django_filters.MultipleChoiceFilter):
 
 
 class MultiValueUUIDFilter(django_filters.UUIDFilter, django_filters.MultipleChoiceFilter):
-    field_class = multivalue_field_factory(forms.UUIDField)
+    field_class = MultiValueCharField
```

Comment:
I don't fully understand why this wasn't working or how this change fixes it. Are the other uses of `multivalue_field_factory` above going to show the same problem? If not, why not?